### DISCUSSION TITLE
can't import audio file in Parcel 2

### DISCUSSION THREAD

**Discussion by ELI7VH** [parcel-bundler/parcel#6209] [Q&A]
the error: 
```
Error: No transformers found for /.../src/audio/eli7vh-tezos-till-i-bezos-final.wav.
```
also can somebody help me figure out how to import audio files using the `import` syntax

**Comment by devongovett** [comment]
If you want to import it as a url, you can use the `url:` prefix. See https://v2.parceljs.org/configuration/plugin-configuration/#predefined-(offical)-named-pipelines

**Reply by karltaylor** [reply]
> If you want to import it as a url, you can use the `url:` prefix. See https://v2.parceljs.org/configuration/plugin-configuration/#predefined-(offical)-named-pipelines

Sadly this link goes to a 404 now

**Reply by adamreisnz** [reply]
Any updated URL for this help article?

**Reply by adamreisnz** [reply]
The solution is to create a `.parcelrc` file with something like this:

```json
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.{au,wav,mp3}": ["@parcel/transformer-raw"]
  }
}
```

or:

```json
{
  "extends": "@parcel/config-default",
  "transformers": {
    "url:*": ["@parcel/transformer-raw"]
  }
}

```

**Reply by mischnic** [reply]
Or use `new URL()`: https://parceljs.org/languages/javascript/#url-dependencies

**Reply by adamreisnz** [reply]
This actually doesn't seem to work.

I'm defining the URL's in one file so that they can be configurable:

```js
  audio: {
    move: new URL('../audio/move.wav', import.meta.url),
    capture: new URL('../audio/capture.wav', import.meta.url),
  },
```

And in another file they are used:

```js
  for (const key of Object.keys(audio)) {
      if (!audio[key]) {
        continue
      }
      const audioElement = document.createElement('audio')
      audioElement.src = audio[key]
      element.appendChild(audioElement)
      this.audioElements[key] = audioElement
    }
```

This however, doesn't seem to work.

If I instead change the latter to:

```js
      audioElement.src = new URL('../audio/move.wav', import.meta.url)
```

Then it works correctly. The files are in the same relative location to the audio, so it's not a path issue.
Not sure if I'm doing something wrong, but I might stick with the other approach as that works as expected.

**Reply by adamreisnz** [reply]
I ended up handling it differently, so that I could use new URL syntax after all, thanks

**Reply by mischnic** [reply]
Strange, that should be working just fine. The only reason why it might not if this isn't the global `URL`  object but you imported it from somewhere.

**Reply by jamesshapiro** [reply]
Was having issues loading the "use-sound" hook into Parcel and the steps @adamreisnz suggested above helped, but I also had to do 1-2 other things. Linking for anyone running across similar issues: https://stackoverflow.com/questions/78337527/how-to-get-the-use-sound-hook-working-with-parcel

**Comment by adamreisnz** [comment]
No I didn't import it
